manifests/h/HandBrake/HandBrake/1.8.1/HandBrake.HandBrake.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: HandBrake.HandBrake
+PackageVersion: 1.8.1
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+ProductCode: HandBrake
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/HandBrake/HandBrake/releases/download/1.8.1/HandBrake-1.8.1-x86_64-Win_GUI.exe
+  InstallerSha256: F80829D30029BA255675929587F2B6665DE2790E52B24845B92D1427C8893264
+- Architecture: arm64
+  InstallerUrl: https://github.com/HandBrake/HandBrake/releases/download/1.8.1/HandBrake-1.8.1-arm64-Win_GUI.exe
+  InstallerSha256: 78E5E2E7061136EB0066BBFA3759A27A45A20CD3280FA30E5FA369BD06A8D22E

```

```suggestion
  InstallerSha256: 2A3AC4406A6F3D85DE2F5DB2CDB8D83B76D7A33553AC2BEBD587972280613CA9
```

Re-hash the installer file with _Komac_.